### ISSUE TITLE
the value loaded is different with different pytorch versions

### ISSUE THREAD

**Issue by luhc15** [pytorch/pytorch#15055]
I loaded the pretrained weights with  different pytorch version, but got slight difference. for example, the value i loaded with pytorch 0.3.1 is " features.denseblock2.denselayer1.conv.1.weight"
 ( 0 , 0 ,.,.) =
  7.6091e-03

( 0 , 1 ,.,.) =
  2.0336e-03

( 0 , 2 ,.,.) =
 -5.3382e-03
    ...
. whie the value of the same keys is slightly different with pytorch 0.4.1, which is 
tensor([[[[ 0.0076]],

         [[ 0.0020]],

         [[-0.0053]],

         ...,

the command I used is  torch.load.
why did this happen??  any one encountered this problem?

**Comment by albanD** [comment]
Hi,

These are actually the same, they are just printed differently because we changed the way Tensors are printed between these versions.

**Comment by luhc15** [comment]
thanks for your reply, but I  found  some values in 0.41 are:
         [[-0.0375]],

         [[-0.0290]],

         [[ 0.0472]]],


        [[[ 0.0104]],

         [[ 0.0203]],

         [[-0.0040]],

         ...
****************************
however  in  the 0.31 are 
( 1 ,189,.,.) =
 -3.7497e-02

( 1 ,190,.,.) =
 -2.8977e-02

( 1 ,191,.,.) =
  4.7178e-02
      ⋮

( 2 , 0 ,.,.) =
  1.0442e-02

( 2 , 1 ,.,.) =
  2.0312e-02

( 2 , 2 ,.,.) =
 -3.9759e-03
********************
they are really the same rather than just printed differently??  because the bit that<5 is default to 0, and >5 is add 1to former bit.

**Comment by albanD** [comment]
Yes that's rounding during the printing.
To be sure. You can extract one number as a python number with indexing and `.item()` and print the python number. You'll see that the python numbers are the same (as python float printing is the same in both cases).

**Comment by luhc15** [comment]
yeah, you are right, thank you.  I got worse results using 0.4.1 with the model converted from 0.3.1 and checked the parameters. the parameters are totally same.  the only difference now is Batchnorm2d, which additional num_batches_tracked is added in 0.4.1. is  there some effects with this additional parameter?
